ca/ca.go

```diff
@@ -33,6 +34,8 @@ type CAImpl struct {
 	ocspResponderURL string
 
 	chains []*chain
+
+	certLifetime int
```

Comment:
Make this a `uint` so you don't have to check for `> 0` later on.

Reply:
Yeah, I tried using `uing` in #362 I believe and something broke in that PR.. So I didn't bother looking at the other PRs to be honest. Should probably work here.